12-rest/classworks/Gloria_Loi/index.js

```diff
@@ -0,0 +1,211 @@
+const Koa = require('koa');
+const Router = require('koa-router');
+const bodyParser = require('koa-bodyparser');
+const crypto = require('crypto');
+const fs = require('fs');
+const koaJwt = require('koa-jwt');
+const jwt = require('jsonwebtoken');
+const fetch = require('node-fetch');
+
+const public = Router();
+const private = Router();
+
+const mySecret = 'super-mega-ochen-secret';
+
+const users = JSON.parse(fs.readFileSync('./database/file.json'));
+let user;
```

Comment:
it should not be a global variable. imagine someone is logging in and at this moment another user trie to log in and can rewrite this var. better create a user in every log in request